src/main/features/dashboard/views/macro/claimStatus/claimFullyAdmitted.njk

```diff
@@ -97,15 +97,17 @@
       <h2 class="heading-small">{{ t('What to do if they don’t pay') }}</h2>
       <p> {{ t('If the defendant has not paid you by {{ paymentDate }}, you can request a County Court Judgment.', { paymentDate: paymentDate}) }} </p>
     {% endif %}
-  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE
+          and (not claim.settlement or claim.settlement and not claim.settlement.isOfferResponded()) %}
```

Comment:
Can you please put bracket or separate out or statement e.g . ((not claim.settlement) or (claim.settlement and not claim.settlement.isOfferResponded()))